models/spring-ai-ollama/src/test/java/org/springframework/ai/ollama/OllamaChatModelMultimodalIT.java

```diff
@@ -53,7 +53,7 @@ class OllamaChatModelMultimodalIT {
 	@Container
 	static OllamaContainer ollamaContainer = new OllamaContainer("ollama/ollama:0.1.32");
```

Comment:
Consider updating this image to version "0.2.8" as well

Reply:
I updated and it seems there is a regression, the description of the image returned now is totally wrong.  I'll merge the PR with the updated versions but we need to circle back and manually test.